jsapp/js/components/processing/analysis/responseForms/responseWrapper.component.tsx

```diff
@@ -97,6 +99,12 @@ export default function ResponseWrapper(props: ResponseWrapperProps) {
     }
   }
 
+  function handleClear() {
+    if (typeof props.clearable === 'function') {
+      props.clearable()
+    }
```

Comment:
Why runtime-check for a function, it's already typed as function. `props?.clearable()` would be much cleaner